index.js

```diff
@@ -837,6 +837,8 @@ function findRefs (obj, options) {
  *
  * @returns {Promise} a promise that resolves a {@link module:JsonRefs~RetrievedRefsResults}
  *
+ * @throws {Error} if the provided `location` argument is not a string
```

Comment:
I'm not sure `@throws` is accurate here as errors aren't thrown, as in you wrap a call to `JsonRefs.findRefsAt` in a `try/catch`.  Things are a little muddy when it comes to how to use jsdoc and `Promise` objects.  Let me do some digging.  *(Same goes to the following, similar changes.)*